src/sdk/model/o365/o365-job.ts

```diff
@@ -245,6 +246,18 @@ export class O365Job extends Entity {
     });
   }
 
+  /**
+   * Modify an O365 backup job
+   * @param {O365ModifyJobRequest} request
+   * @returns {Promise<O365Job>}
+   */
```

Comment:
add `  /* istanbul ignore next: autogenerated */` right above the method signature to help keep the test coverage % from dropping

Reply:
Done!